install.sh

```diff
@@ -92,7 +102,10 @@ eval "\$(/opt/azurehpc/slurm/venv/bin/register-python-argcomplete azslurm)" || e
 EOF
 fi
 
-which jetpack || exit 0
```

Comment:
I think earlier we were detecting if jetpack did not exist. Now it seems we are assuming that the script has been passed with the right options. which is great but I think we should still detect if jetpack is present and set the `NO_JETPACK` variable accordingly?